drop-transfer/src/ws/server/mod.rs

```diff
@@ -351,6 +365,7 @@ async fn handle_rejection(
 struct RunContext<'a> {
     logger: &'a slog::Logger,
     state: Arc<State>,
+    refresh_trigger: &'a tokio::sync::watch::Receiver<()>,
```

Comment:
Do we use this field?

Reply:
`init_manager` function uses it to spawn check tasks